pythonforandroid/recipes/mplfinance/__init__.py

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
Typically this recipe as well as `pandas` could have been added in another dedicated PR, right?
This two recipe alone are simple enough to review, but in general if should try to reduce huge PR when possible to reviewers don't get scared/discouraged by the size of it or lose focus during the review. Also if large PR receive a lot of comments then the contributor might as well get discouraged or simply side tracked and drop it.
By the way line one is empty :smile: